src/styles/reset.less

```diff
@@ -11,13 +13,99 @@
  * make sure we're resetting all spacing elements.
  *
  */
+
+  // Light Theme Variables
+ .light-theme {
+  --theme: light;
+  --theme-primary-1: @light-blue-100; 
+  --theme-primary-2: @light-blue-200; // profile card borders, buttons, nav bar 
+  --theme-primary-3: @light-blue-300;
+  --theme-primary-4: @light-blue-400;// circle background, link hover, progress bar
+  --theme-primary-5: @light-blue-500; // event card links, arrow hover
+  --theme-primary-6: @light-blue-600; // event card borders
+
+  --theme-success-1: @green-100;
+  --theme-success-2: @green-200;
+
+  --theme-gold: @light-gold;
+  --theme-silver: @light-silver;
+  --theme-bronze: @light-bronze;
+
+  --theme-danger-1: @red;
+
+  --theme-invert-percent: 0%;
+
+  --theme-disabled: @gray-600; // disabled elements
+
+  --theme-background: @white; // page background
+
+  --theme-surface-1: @gray-200; // table rows
+  --theme-surface-2: @gray-300; // nav bar background
+
+  --theme-accent-line-1: @gray-900; // heavy accent lines and gray borders
+  --theme-accent-line-2: @gray-400; // light accent lines and gray borders
+  --theme-accent-line-1-transparent: rgba(@gray-900, 0.5); // light accent lines
+  --theme-accent-line-2-transparent: rgba(@gray-400, 0.5); // light accent lines
+
+  --theme-text-on-background-1: @gray-900; // text on primary titles and input borders
+  --theme-text-on-background-2: @gray-700; // text on primary subtitles (LVL, Dates)
+
+  --theme-text-on-surface-1: @gray-900; // dark text on sidebar
+  --theme-text-on-surface-2: @gray-500; // light titles on sidebar
+
+  background-color: var(--theme-background);
+  transition: background 0.3s ease;
```

Comment:
imo specifically transition `background-color` since that's what youre defining

could look into making a less variable / mixin / w/e for the default transition duration / easing